store/store.go

```diff
@@ -1254,6 +1259,37 @@ func (s *Store) Sections(ctx context.Context, user *auth.UserState) ([]string, e
 	return sectionNames, nil
 }
 
+// Categories retrieves the list of available store categories.
+func (s *Store) Categories(ctx context.Context, user *auth.UserState) ([]string, error) {
+	reqOptions := &requestOptions{
+		Method:         "GET",
+		URL:            s.endpointURL(categoriesEndpPath, nil),
+		Accept:         halJsonContentType,
```

Comment:
the accept seems wrong here as the result is not HAL JSON anymore as that was used by v1, instead I think this needs to set APILevel: apiV2Endps though